spec/features/user_views_list_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'User views list' do
+  scenario  "and see's the list's books" do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.